provd/proto/privacy.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package privacy;
+
+service PrivacyService {
+    rpc SetPrivacySettings (SetPrivacySettingsRequest) returns (Emtpy) {}
+}
+
+message Empty {}
+
+message SetPrivacySettingsRequest {
+    bool locationEnabled = 1;
+    bool reportingEnabled = 2;
```

Comment:
I think we can do better with both names here. For example, is 'reportingEnabled' about ongoing bug reporting, one off installation metrics, or something else entirely?

Reply:
Looking at the flow screenshots / dart pages, I see that although the model includes the reporting toggle, it is not actually implemented in page yet.